apps/barrel_httpc/src/barrel_httpc.erl

```diff
@@ -188,13 +192,21 @@ connect(Url) ->
   Meta :: meta(),
   Res :: {ok, Doc, Meta} | {error, not_found} | {error, any()}.
 get(Conn, DocId, Options0) ->
-  {Headers, Options1} = headers(Options0),
-  Url = barrel_httpc_lib:make_url(Conn, [<<"docs">>, DocId], Options1),
+  {WithAttachment, Options1} = maybe_with_attachments(Options0),
+  {Headers, Options2} = headers(Options1),
+  Url = barrel_httpc_lib:make_url(Conn, [<<"docs">>, DocId], Options2),
   case request(Conn, <<"GET">>, Url, Headers, <<>>) of
     {ok, 200, RespHeaders, JsonBody} ->
       Doc = jsx:decode(JsonBody, [return_maps]),
+      {Attachments, DocWithoutAttachment} = maybe_take(<<"_attachments">>, Doc, []),
```

Comment:
Are there case where you will need to pass a different value to Attachments? if not i would skip the default argument.